meta/main.yml

```diff
@@ -5,14 +5,14 @@ galaxy_info:
   description: An Ansible role for installing Open Trip Planner.
   company: Azavea Inc.
   license: Apache
-  min_ansible_version: 1.5
+  min_ansible_version: 2.0
   platforms:
   - name: Ubuntu
     versions:
-    - trusty
+    - xenial
```

Comment:
It's required to use openjdk 8, which is needed to upgrade OTP from the version it's at now. I imagine we won't want to use OTP versions <= what is is now for this or any future projects.